backend/apps/feature_requests/signals.py

```
@@ -0,0 +1,92 @@
+        vote_type='upvote'
+    ).count()
+    feature.downvotes = Vote.objects.filter(
+        feature_request=feature,
+        vote_type='downvote'
+    ).count()
+    feature.total_votes = feature.upvotes + feature.downvotes
+    
+    # Calculate weighted votes
+    weighted = Vote.objects.filter(
+        feature_request=feature
+    ).aggregate(total=Sum('weight'))['total'] or 0
```

Ruff (0.15.20)

[error] Undefined name `Sum`

(F821)